js/homeview.js

```diff
@@ -233,7 +233,7 @@ enyo.kind({
 
 	localize: function() {
 		if (this.otherview && this.otherview.localize) {
-			this.otherview.localize();
+			this.otherview.localize;
```

Comment:
Very strange fix: removing parenthesis is equivalent to don't call the function at all.